integration/shareddata/scalars.go

```diff
@@ -24,36 +24,37 @@ import (
 )
 
 const (
-	doubleMaxPrec = float64(1<<53 - 1) // 9007199254740991.0:    largest double values that could be represented as integer exactly
-	doubleBig     = float64(1 << 61)   // 2305843009213693952.0: some number larger than safe integer (doubleBig+1 == doubleBig)
-	longBig       = int64(1 << 61)     // 2305843009213693952:   same as doubleBig but integer
+	doubleMaxPrec float64 = float64(1<<53 - 1) // 9007199254740991.0:    largest double values that could be represented as integer exactly
```

Comment:
There is no need to specify type twice – it looks intentional, but actually does nothing and is confusing